src/components/Results.vue

```diff
@@ -3,8 +3,8 @@
 
     <slot name="header"></slot>
 
-    <slot v-for="result in results" :result="result">
-      Result 'objectID': {{ result.objectID }}
+    <slot v-for="(result, index) in results" :result="result" :index="index">
+      Result key ( {{ index }} ) 'objectID': {{ result.objectID }}
```

Comment:
Yeah. it might be confusing to see "key" as well (I used it at first but it's a reserved word ^^)